.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: go
+
+go:
+  - "1.10"
```

Comment:
this is an old version of go

Reply:
It comes with ubuntu 19.04 by default. Also, the code survived jumping from 1.5 to 1.10 without any problems, so I think it is not very critical.

Reply:
i'm thinking about creating a release at the `fzp` repository to use a specific version for the ci tests.
instead of running `go get ...` we can do a `wget to fetch the binary for the travis-ci machine.

then we cannot break the ci for fritzing parts if there are ongoing changes at the `fzp` cli tool.

Reply:
Why aren't we running this in Docker? We could then use what ever version of Go we want.

Reply:
docker would be overkill, and quite inconvenient to work with. Are there any issues I overlooked? 1.10 seems to work quite smooth, but of course we can add 1.12 to the CI if you want.

Reply:
It would make more sense to test different go versions on the fzp and fzb repos, though.

Reply:
i think it is the best to use a prebuild binary for the tests at the fritzing-parts repo. then we can change the travis language to `shell` or something. 

@KjellMorgenstern testing `fzp` and `fzb` against different go versions is a good idea. i already setup tests for go 1.5, 1.6 and the latest. need to update the travis config to test the other versions also